### DISCUSSION TITLE
QuickSplash and Wallpaper Manager Integration

### DISCUSSION THREAD

**Discussion by Itz-Abhishek-Tiwari** [prasanthrangan/hyprdots#1231] [Ideas]
I was thinking, since you guys are building a hyperland setup, which is like a DE in itself, why don't we add a quick-splash that changes according to the theme? Also, a wallpaper manager that can display wallpapers from users and all themes in one place (and wallpapers sorted by theme should also be included)

**Comment by kRHYME7** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/d0b00be6-779e-479d-989c-313d33c30d40)

We already got something that kind of meets the description. What do you mean a Fancier looking one?

**Reply by Itz-Abhishek-Tiwari** [reply]
yep like a kde wallpaper picker but build with rofi if possible ??

**Reply by Itz-Abhishek-Tiwari** [reply]
like here i have to paste the wallpaper and then build cache so that it will show on my rofi menu what i want is that i can paste my wallpaper here but the cache building process should be automated or should not be required to do.(I generally download 20 wallpaper in one go and use them for like a 1 or 2 months then do again).

_sorry for bad English_
